packages/bemlinter/README.md

```diff
@@ -85,6 +85,22 @@ default: `[]`
 }
 ```
 
+### snapshot (option)
+
+A boolean if you want to monitor the quality tendency instead of targeting absolutely zero error
+
+:information_source: You also can set a file path of your snapshot
```

Comment:
And maybe precise that the output will be by default written in `.bemlinter-snap` when snapshot option is `true`